core/client/src/backend.rs

```diff
@@ -34,6 +34,25 @@ pub type StorageCollection = Vec<(Vec<u8>, Option<Vec<u8>>)>;
 /// In memory arrays of storage values for multiple child tries.
 pub type ChildStorageCollection = Vec<(Vec<u8>, StorageCollection)>;
 
+/// Import operation wrapper
+pub struct ClientImportOperation<
```

Comment:
What is the reason for the multiple tuples nested (indirectly) in this struct?

Reply:
this, too, didn't actually change but just move the struct from client to backend, which it a) is more closely related to and b) can be used here without creating a circle dependency between backend<->client. Code didn't change and unfortunately I can't really attest to the actual nature or decision upon why this is structured the way it is - but I consider that as out of scope for this PR.